person_capture/gui_app.py

```
@@ -2687,6 +2709,35 @@ def _cooldown_frames() -> int:
+                        os.environ.pop("OPENCV_FFMPEG_CAPTURE_OPTIONS", None)
+                    else:
+                        os.environ["OPENCV_FFMPEG_CAPTURE_OPTIONS"] = prev_ffmpeg_opts
+                if hasattr(reader, "isOpened") and not reader.isOpened():
+                    reader.release()
+                    return cv2.VideoCapture(video_path)
+                pos0 = int(reader.get(cv2.CAP_PROP_POS_FRAMES) or 0)
+                ok = bool(reader.grab())
+                try:
+                    reader.set(cv2.CAP_PROP_POS_FRAMES, pos0)
+                except Exception:
+                    pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)